doc/Tests_XML_Definition_and_Sample.md

```diff
@@ -33,6 +44,186 @@ Example:
+        </description>
+        <specs>
+          <spec>
+            <spec_assertion category="Tizen W3C API Specifications" interface="HTMLAudioElement" section="Media" specification="HTML5 The audio element" usage="true"/>
+            <spec_url>http://www.w3.org/TR/2012/WD-html5-20121025/media-elements.html#htmlmediaelement</spec_url>
+            <spec_statement/>
+          </spec>
+        </specs>
+      </testcase>
+    </set>
+  </suite>
+</test_definition>
+```
+#### "wrt" type
+
+The test contains test application, need testkit-lite to support install/uninstall test Apps by test set name. 
+
+Example:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="./testcase.xsl"?>
+<test_definition>
+  <suite category="Runtime" name="wrt-rt-linux-tests">
+  ...
+    <set name="connectsrccrossoriginmultixmlhttprequestallowedtwo" type="wrt">
+     <testcase component="Crosswalk WRT/CSP" execution_type="auto" id="connectsrccrossoriginmultixmlhttprequestallowedtwo" priority="P1" purpose="Check if user agent is able to open second allowed external resource by xhr when connect-src is cross-origin." status="approved" type="compliance">
+        <description>
+          <pre_condition/>
+          <post_condition/>
+          <steps>
+            <step order="1">
+              <step_desc>Check if user agent is able to open second allowed external resource by xhr when connect-src is cross-origin.</step_desc>
+              <expected>PASS</expected>
+            </step>
+          </steps>
+          <test_script_entry test_script_expected_result="0">test_index.html</test_script_entry>
+        </description>
+      </testcase>
+    </set>
+  </suite>
+</test_definition>
+```
+
+#### "qunit" type
+
+The qunit style test
+
+Example:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="./testcase.xsl"?>
+<test_definition>
+  <suite category="SIMD" name="webapi-simd-nonw3c-tests">
+    <set name="webapi-simd-nonw3c-tests" type="qunit">
+      <testcase component="Supplementary APIs/Experimental/SIMD" execution_type="auto" id="Float32x4_constructor" priority="P1" purpose="Check Float32x4 constructor (0, 2, 2)" status="approved" subcase="6" type="compliance">
+        <description>
+          <test_script_entry>/opt/webapi-simd-nonw3c-tests/simd/ecmascript_simd/src/index.html?testNumber=1</test_script_entry>
+        </description>
+        <specs>
+          <spec>
+            <spec_assertion category="SIMD" element_name="Float32x4" element_type="method" interface="SIMD" section="SIMD" specification="SIMD"/>
+            <spec_url>https://github.com/johnmccutchan/ecmascript_simd/blob/master/README.md</spec_url>
+            <spec_statement/>
+          </spec>
+        </specs>
+      </testcase>
+    </set>
+  </suite>
+</test_definition>
+```
+#### "nodeunit" type
+
+The test load with node js unit test runner
```

Comment:
=> Tests written in [nodeunit](https://github.com/caolan/nodeunit) style, which need to load the nodeunit.js module.